stories/documentation/forms/fields/rich-text/angular/rich-text-input.stories.ts

```diff
@@ -0,0 +1,82 @@
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FormFieldComponent } from '@lucca-front/ng/form-field';
+import {
+	ClearFormatComponent,
+	HeadingsComponent,
+	LinkComponent,
+	ListStyleToolbarComponent,
+	provideLuRichTextMarkdownFormatter,
+	RichTextInputComponent,
+	TextStyleComponent,
+	TextStyleToolbarComponent,
+} from '@lucca-front/ng/forms/rich-text-input';
+import { Meta, moduleMetadata, StoryObj } from '@storybook/angular';
+import { DividerComponent } from '@lucca-front/ng/divider';
+import { cleanupTemplate, generateInputs } from 'stories/helpers/stories';
+import { StoryModelDisplayComponent } from 'stories/helpers/story-model-display.component';
+
+export default {
+	title: 'Documentation/Forms/Fields/RichTextInput/Angular',
+	decorators: [
+		moduleMetadata({
+			imports: [
+				RichTextInputComponent,
+				HeadingsComponent,
+				LinkComponent,
+				ClearFormatComponent,
+				TextStyleToolbarComponent,
+				ListStyleToolbarComponent,
+				TextStyleComponent,
+				FormFieldComponent,
+				FormsModule,
+				ReactiveFormsModule,
+				BrowserAnimationsModule,
+				StoryModelDisplayComponent,
+				DividerComponent,
+			],
+			providers: [provideLuRichTextMarkdownFormatter()],
+		}),
+	],
+	argTypes: {
+		example: {
+			control: {
+				type: 'text',
+			},
+		},
+	},
+} as Meta;
+export const Basic: StoryObj<RichTextInputComponent & { disabled: boolean; example: string } & FormFieldComponent> = {
+	render: (args, { argTypes }) => {
+		const { example, ...inputArgs } = args;
+		return {
+			props: { example },
+			template: cleanupTemplate(`<lu-form-field label="toto">
+	<lu-rich-text-input
+	${generateInputs(inputArgs, argTypes)}
+		[(ngModel)]="example">
+				<div class="u-width100% u-displayFlex u-justifyContentSpaceBetween pr-u-gap100">
```

Comment:
```suggestion
				<div class="u-width100% u-displayFlex u-justifyContentSpaceBetween pr-u-gap100">
```
Proposer une classe interne à la place